### DISCUSSION TITLE
Adjust default zoom on Issue inspection

### DISCUSSION THREAD

**Discussion by germzx** [sn4k3/UVtools#499] [Ideas]
I might be using the Issue function wrong, but after detecting issues and clicking through, I find that the default zoom is too far in, requiring me to click > unzoom > decide what I want to do > usually rezoom, on every task. It would be nice if I could adjust the default zoom to get a better sense of where each highlighted feature is in respect to the rest of the layer to make the best decision of how to fix it. 

A much harder solution I'm sure would be a picture in picture type zoom. where most of your screen is the zoomed in feature in question, while in a corner could be the entire layer fit to width of a small box, perhaps with a highlight around where your zoom in is actually looking.

**Comment by sn4k3** [comment]
You can adjust settings:
![image](https://user-images.githubusercontent.com/113281/176041832-208fda79-1bc5-46ab-9a14-aec05377af3b.png)

If you hold ALT key while navigate issues it will not auto-zoom or will depending on your setting (Alt will temporary invert your zoom setting)
